test/teiserver/battle/split_one_chevs_internal_test.exs

```diff
@@ -4,30 +4,50 @@ defmodule Teiserver.Battle.SplitOneChevsInternalTest do
   Can run tests in this file only by
   mix test test/teiserver/battle/split_one_chevs_internal_test.exs
   """
-  use Teiserver.DataCase, async: true
+  use ExUnit.Case
   @moduletag :balance_test

```

Comment:
Changes to this file are just formatting and also since we don't hit the db, we use ExUnit instead